src/i18n-module-wordpressorg.php

```diff
@@ -15,16 +15,35 @@ class Yoast_I18n_WordPressOrg_v2 {
 	/**
 	 * Constructs the i18n module for wordpress.org. Required fields are the 'textdomain', 'plugin_name' and 'hook'
 	 *
-	 * @param array $args The settings for the i18n module.
+	 * @param array $args                   The settings for the i18n module.
+	 * @param bool $show_translation_box    Whether the translation box should be shown.
 	 */
-	public function __construct( $args ) {
+	public function __construct( $args, $show_translation_box = true ) {
 		$args = $this->set_defaults( $args );
 
-		$this->i18n = new Yoast_I18n_v2( $args );
+		$this->i18n = new Yoast_I18n_v2( $args, $show_translation_box );
 		$this->set_api_url( $args['textdomain'] );
 	}
 
 	/**
+	 * Returns the i18n_promo message from the i18n_module.
+	 *
+	 * @return string The i18n promo message.
+	 */
+	public function promo_message() {
```

Comment:
It would be nice to make this method a wrapper method:

```php
public function get_promo_message() {
    return $this->i18n->get_promo_message();
}
```

The implementation in the `Yoast_I18n_v2` can be:
```php
public function get_promo_message() {
    if ( $this->is_admin_in_other_language() ) {
        return $this->promo_message();
    } 

    return '';
}
```